pages/doc/opentelemetry_java_tutorial.md

```diff
@@ -1,10 +1,97 @@
 ---
-title: Manually Instrument Java Application with OpenTelemetry
-keywords: integrations
-layout: readme
-source: opentelemetry-examples/java-example
+title: Auto Instrument Java Application with OpenTelemetry
 tags: [tracing]
 sidebar: doc_sidebar
 permalink: opentelemetry_java_app_tutorial.html
-summary: Instrument your Java application to send OpenTelemetry trace data to Tanzu Observability. 
+summary: Auto instrument your Java application to send OpenTelemetry trace data to our service. 
 ---
+
+This guide shows you how to auto instrument your Java application using the OpenTelemetry Java agent. and send data to our service. 
+
+## Prerequisites
+
+* A VMware Aria Operations for Applications (formerly known as Tanzu Observability by Wavefront) account to visualize and monitor your application health. If you don’t have one already, you can sign up on [our website](https://www.vmware.com/products/aria-operations-for-applications.html). 
+* Docker to run the Wavefront proxy. 
+* Java 11 or higher
+* Maven
+
+
+## Install the Wavefront Proxy
+
+Follow these steps to install the Wavefront proxy using Docker. See [Install a Proxy](https://docs.wavefront.com/proxies_installing.html#install-a-proxy) to find other options for installing the proxy on your environment.
+
+1. Make sure you have **Proxies** permission by going to `https://{cluster}.wavefront.com/userprofile/groups`. A checkbox next to **Proxies** indicates you have permission. If you don’t see a checkbox next to **Proxies**, your administrator needs to add this permission to your user account.
```

Comment:
1. Make sure that you have  the **Proxies** permission. 
    a. Click the gear icon on the toolbar and select your username.
    b. On the **Groups, Roles & Permissions** tab, verify that the **Proxies** check box is selected.
    c. If you don’t see the check box next to **Proxies** selected, ask a user with the **Accounts**  permission to grant you with the **Proxies** permission.